### PR TITLE
Update seed testing instruction.

### PR DESCRIPTION
**Pull request by goodov**

### CODE REVIEW

**Comment by goodov** [comment]
cc @brave/qa-team 

can you please proof-read the instruction about testing the seed and improve it if possible? Maybe you have a better naming internally for "the debug menu" on android/ios respectively.

File: seed/seed.json [hunk 1]
```diff
@@ -2948,7 +2948,7 @@
                             "Playlist"
                         ]
                     },
-                    "name": "Enabled",
+                    "name": "Enabledd",
```
Comment (stephendonner) [inline seed/seed.json:2951]: What's this about?  Is it an intentional change, or a typo?

The rest looks good to me for desktop 👍

**Review by kjozwiak — APPROVED** [review]
LGTM other than the issue that @stephendonner pointed out re: `Enabledd`. Instructions should be good enough. At this point, most of QA should be able to figure out how to verify using the new method but I'll point them to our verifications via https://github.com/brave/brave-core/pull/24604#issuecomment-2231943968 which should have enough context/information.

File: seed/seed.json (hunk 1 above)
Comment (goodov) [reply seed/seed.json:2951]: it's a test change to trigger the seed generation.

**Comment by goodov** [comment]
> LGTM other than the issue that @stephendonner pointed out re: `Enabledd`. Instructions should be good enough. At this point, most of QA should be able to figure out how to verify using the new method but I'll point them to our verifications via [brave/brave-core#24604 (comment)](https://github.com/brave/brave-core/pull/24604#issuecomment-2231943968) which should have enough context/information.

The point here is so any developer who sees variations the first time can figure this on its own, it's not only about QA.

**Comment by kjozwiak** [comment]
> > LGTM other than the issue that @stephendonner pointed out re: `Enabledd`. Instructions should be good enough. At this point, most of QA should be able to figure out how to verify using the new method but I'll point them to our verifications via [brave/brave-core#24604 (comment)](https://github.com/brave/brave-core/pull/24604#issuecomment-2231943968) which should have enough context/information.
> 
> The point here is so any developer who sees `brave-variations` for the first time can figure this on their own, it's not only about QA.

yup, makes sense 👍

**Pull request merged**